docs/services/motion/_index.md

```diff
@@ -73,840 +73,7 @@ For examples showing how to construct the `ResourceName`, see the code samples b
-You can use the `supplemental_transforms` argument to augment the machine's existing frame system with supplemental frames.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
-
-**Parameters:**
-
-- `component_name` ([ResourceName](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.ResourceName)): The `ResourceName` of the piece of the machine whose pose is returned.
-
-- `destination_frame` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)):
-  The name of the frame with respect to which the component's pose is reported.
-
-- `supplemental_transforms` ([Optional\[List\[Transforms\]\]](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Transform)) (_optional_): A list of `Transform` objects.
-  A `Transform` represents an additional frame which is added to the machine's frame system.
-  It consists of the following fields:
-
-  - `pose_in_observer_frame`: Provides the relationship between the frame being added and another frame.
-  - `physical_object`: An optional `Geometry` can be added to the frame being added.
-  - `reference_frame`: Specifies the name of the frame which will be added to the frame system.
-
-  When `supplemental_transforms` are provided, a frame system is created within the context of the `GetPose` function.
-  This new frame system builds off the machine's frame system and incorporates the `Transform`s provided.
-  If the result of adding the `Transform`s results in a disconnected frame system, an error is thrown.
-
-- `extra` (Mapping[str, Any]) (_optional_): A generic struct, containing extra options to pass to the underlying RPC call.
-
-**Returns:**
-
-- [(PoseInFrame)](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame): The pose of the component.
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/motion/index.html#viam.services.motion.MotionClient.get_pose).
-
-The following code example gets the pose of the tip of a [gripper](/components/gripper/) named `my_gripper` which is attached to the end of an arm, in the "world" `reference_frame`:
-
-```python {class="line-numbers linkable-line-numbers"}
-from viam.components.gripper import Gripper
-from viam.services.motion import MotionClient
-
-# Assume that the connect function is written and will return a valid machine.
-robot = await connect()
-
-motion = MotionClient.from_robot(robot=robot, name="builtin")
-gripperName = Gripper.get_resource_name("my_gripper")
-gripperPoseInWorld = await motion.get_pose(component_name=gripperName,
-                                           destination_frame="world")
-```
-
-For a more complicated example, take the same scenario and get the pose of the same gripper with respect to an object situated at a location (100, 200, 0) relative to the "world" frame:
```

Comment:
This feels like it should also remain - added as a secondary example to the python docs

Reply:
Sure, re-added it back 👍